charts/newrelic-infrastructure-beta/values.yaml

```diff
@@ -0,0 +1,393 @@
+# Default values for newrelic-infrastructure.
+nameOverride: ""
+fullnameOverride: ""
+
+enableLinux: true
+enableWindows: false
+windowsOsList:
+  # - version: 1809
+  #   imageTag: 2.2.0-windows-1809-alpha
+  #   buildNumber: 10.0.17763
+  # - version: 1909
+  #   imageTag: 2.2.0-windows-1909-alpha
+  #   buildNumber: 10.0.18363
+  - version: 2004
+    imageTag: 2.2.0-windows-2004-alpha
+    buildNumber: 10.0.19041
+
+images:
+  pullSecrets: []
+  forwarder:
+    pullPolicy: IfNotPresent
+    tag: 1.20.5
+    repository: newrelic/k8s-events-forwarder
+  agent:
+    pullPolicy: IfNotPresent
+    tag: 2.7.6
+    repository: newrelic/infrastructure-bundle
+  scraper:
+    pullPolicy: IfNotPresent
+    tag: 0.1.1
+    repository: newrelic/nri-kubernetes
+
+controlPlane:
+  enabled: false
```

Comment:
why false by default?

Reply:
don't we want it false by default? should it be true?

Reply:
There can be a weird interaction if it is enabled, set to `daemonSet`, but `privileged` is false. We need to plan this carefully @gsanchezgavier

Reply:
(In any case I think it should be on by default if possible)